src/browser/ui/dom/components/__tests__/ReactDOMInput-test.js

```diff
@@ -299,30 +303,41 @@ describe('ReactDOMInput', function() {
       React.render(<input type="checkbox" checkedLink={link} />, node);
       expect(console.warn.mock.calls.length).toBe(0);
 
-      React.render(
+      ReactTestUtils.renderIntoDocument(
         <input
           type="checkbox"
           checked="false"
           onChange={mocks.getMockFunction()}
-        />,
-        node
+        />
       );
       expect(console.warn.mock.calls.length).toBe(0);
 
-      React.render(
-        <input type="checkbox" checked="false" readOnly={true} />,
-        node
+      ReactTestUtils.renderIntoDocument(
+        <input type="checkbox" checked="false" readOnly={true} />
       );
       expect(console.warn.mock.calls.length).toBe(0);
 
-      React.render(<input type="checkbox" checked="false" />, node);
+      ReactTestUtils.renderIntoDocument(<input type="checkbox" checked="false" />);
       expect(console.warn.mock.calls.length).toBe(1);
+    } finally {
+      console.warn = oldWarn;
+    }
+  });
+
+  it('should warn with checked and no onChange handler with readOnly specified', function() {
+    var oldWarn = console.warn;
+    try {
+      console.warn = mocks.getMockFunction();
+
+      ReactTestUtils.renderIntoDocument(
+        <input type="checkbox" checked="false" readOnly={true} />
+      );
+      expect(console.warn.mock.calls.length).toBe(0);
 
-      React.render(
-        <input type="checkbox" checked="false" readOnly={false} />,
-        node
+      ReactTestUtils.renderIntoDocument(
+        <input type="checkbox" checked="false" readOnly={false} />
       );
-      expect(console.warn.mock.calls.length).toBe(2);
+      expect(console.warn.mock.calls.length).toBe(1);
```

Comment:
There's no functional change to these tests, right? Did I read this correctly? It's just reorganized.

Protip: If there is no functional change and just refactoring, it's good to mention that in the commit message. Makes it easier to review since I otherwise have to look through to find the change.